.goreleaser.yml

```diff
@@ -98,4 +103,31 @@ dockers:
     ids:
       - algolia
       - algolia-linux-arm
-    <<: *docker_defaults
\ No newline at end of file
+    <<: *docker_defaults
+
+chocolateys:
```

Comment:
@clemfromspace, the following files should be updated as you regard.

to understand the available options : https://goreleaser.com/customization/chocolatey/